File: app/src/main/java/poker/fivecardstats/model/Score.java
package poker.fivecardstats.model;

public class Score {
    private Long id;
    private Long userId;
    private String type;
    private int score;

    public Score(Long id, Long userId, String type, int score) {
        this.id = id;
        this.userId = userId;
        this.type = type;
        this.score = score;
    }

    public Long getId() {
        return id;
    }

    public Long getUserId() { return userId; }

    public String getType() { return type; }

    public int getScore() {
        return score;
    }

    /**
     * @return id + " " + userId + " " + type + " " + score
     */
    public String toString() {
        return id + " " + score;
    }




}

File: app/src/main/kotlin/poker/fivecardstats/model/Score.kt
package poker.fivecardstats.model

data class Score(
        val id: Long,
        val userId: Long,
        val type: String,
        val score: Int
) {
    /**
     * @return id + " " + userId + " " + type + " " + score
     */
    override fun toString(): String = "$id $score"
}